templates/vue/src/services/TapestryAPI.js

```diff
@@ -310,6 +310,23 @@ class TapestryApi {
     const response = await this.client.get(url)
     return response
   }
+
+  async getKalturaStatus() {
+    const url = "/kaltura/status"
+    const response = await this.client.get(url)
+    return response
+  }
+
+  async checkKalturaVideo(entryId) {
+    const url = `/kaltura/video/status?entry_id=${entryId}`
+    const response = await this.client.get(url)
+    return response.data
+  }
```

Comment:
```suggestion
  }

```

Reply:
I'm not sure why this has not been marked Outdated by GitHub, but there should be a space between these methods now.